1.7/usage/managing-services/creating.md

```diff
@@ -0,0 +1,222 @@
+---
+post_title: Creating A Universe Package
+nav_title: Creating
+menu_order: 005
+---
+
+You can create your own package for the DC/OS Universe. [Universe](https://github.com/mesosphere/universe) is the DC/OS package repository that contains services like Spark, Cassandra, Jenkins, and many others.  
+
+Packages are published to the Mesosphere Universe repository on GitHub. At their simplest, DC/OS packages are bundled containerized apps that can be deployed onto a cluster. 
+
+## Getting Started
+
+1.  Install these prerequisite tools.
+    
+    -  Install the DC/OS and the DC/OS CLI using this [guide](/docs/1.7/administration/installing/). You must have at least 1 [public agent](https://docs.mesosphere.com/1.7/overview/concepts/#public) node.
+    -  Install [pip](https://pip.pypa.io/en/stable/installing.html#install-pip)
+    -  Install jsonschema:
+    
+       ```bash
+       $ pip install jsonschema
+       ```
+       
+1.  Fork the [Universe](https://github.com/mesosphere/universe) repository, then clone the fork:
+
+    ```bash
+    $ git clone https://github.com/<your-username>/universe
+    ```
+    
+    As you look around the Universe repository you'll notice that the repository (`/repo/packages`) consists of packages arranged alphabetically. Each package consists of 4 JSON files:
+    
+    -  `config.json` -  configuration properties supported by the package, represented as a json-schema.
+    -  `package.json` -  high-level metadata about the package.
+    -  `marathon.json.mustache` - a [mustache](http://mustache.github.io/) template that when rendered will create a [Marathon](http://github.com/mesosphere/marathon) app definition capable of running your service.
+    -  `resource.json` - contains all of the externally hosted resources (e.g. Docker images, HTTP objects and images) that are required to install the application.
+
+1.  Install the Universe repository pre-commit hooks.
+
+    ```bash
+    $ bash scripts/install-git-hooks.sh
+    ```
+
```

Comment:
I don't think pre-commit hook install is necessary. The user has to run build now to even get a universe-server to test their package so they will have to know about it.